server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tservers/TabletServerResource.java

```diff
@@ -0,0 +1,336 @@
+    double currentMajorStdDev = 0;
+    total = new TabletStats(null, new ActionStats(), new ActionStats(), new ActionStats(), 0, 0, 0, 0);
+    HostAndPort address = HostAndPort.fromString(tserverAddress);
+    historical = new TabletStats(null, new ActionStats(), new ActionStats(), new ActionStats(), 0, 0, 0, 0);
+    List<TabletStats> tsStats = new ArrayList<>();
+
+    try {
+      ClientContext context = Monitor.getContext();
+      TabletClientService.Client client = ThriftUtil.getClient(new TabletClientService.Client.Factory(), address, context);
+      try {
+        for (String tableId : Monitor.getMmi().tableMap.keySet()) {
+          tsStats.addAll(client.getTabletStats(Tracer.traceInfo(), context.rpcCreds(), tableId));
+        }
+        historical = client.getHistoricalStats(Tracer.traceInfo(), context.rpcCreds());
+      } finally {
+        ThriftUtil.returnClient(client);
+      }
+    } catch (Exception e) {
+      return null;
+    }
+
+    List<CurrentOperations> currentOps = doCurrentOperations(tsStats);
+
+    if (total.minors.num != 0)
+      currentMinorAvg = (long) (total.minors.elapsed / total.minors.num);
+    if (total.minors.elapsed != 0 && total.minors.num != 0)
+      currentMinorStdDev = stddev(total.minors.elapsed, total.minors.num, total.minors.sumDev);
+    if (total.majors.num != 0)
+      currentMajorAvg = total.majors.elapsed / total.majors.num;
+    if (total.majors.elapsed != 0 && total.majors.num != 0 && total.majors.elapsed > total.majors.num)
+      currentMajorStdDev = stddev(total.majors.elapsed, total.majors.num, total.majors.sumDev);
+
+    ActionStatsUpdator.update(total.minors, historical.minors);
+    ActionStatsUpdator.update(total.majors, historical.majors);
+    totalElapsedForAll += total.majors.elapsed + historical.splits.elapsed + total.minors.elapsed;
+
+    minorStdDev = stddev(total.minors.elapsed, total.minors.num, total.minors.sumDev);
+    minorQueueStdDev = stddev(total.minors.queueTime, total.minors.num, total.minors.queueSumDev);
+    majorStdDev = stddev(total.majors.elapsed, total.majors.num, total.majors.sumDev);
+    majorQueueStdDev = stddev(total.majors.queueTime, total.majors.num, total.majors.queueSumDev);
+    splitStdDev = stddev(historical.splits.num, historical.splits.elapsed, historical.splits.sumDev);
+
+    TabletServerDetailInformation details = doDetails(address, tsStats.size());
+
+    List<AllTimeTabletResults> allTime = doAllTimeResults(majorQueueStdDev, minorQueueStdDev, totalElapsedForAll, splitStdDev, majorStdDev, minorStdDev);
+
+    CurrentTabletResults currentRes = doCurrentTabletResults(currentMinorAvg, currentMinorStdDev, currentMajorAvg, currentMajorStdDev);
+
+    TabletServerSummary tserverDetails = new TabletServerSummary(details, allTime, currentRes, currentOps);
+
+    return tserverDetails;
+  }
+
+  private static final int concurrentScans = Monitor.getContext().getConfiguration().getCount(Property.TSERV_READ_AHEAD_MAXCONCURRENT);
```

Comment:
This should be re-computed to ensure that the Monitor picks up new values for this configuration property.